src/main/utils/menuUtil.ts

```diff
@@ -16,12 +16,14 @@ export function buildContextMenu(
 ): Menu {
     const { export_progress, retry_export, paused } = args;
     const contextMenu = Menu.buildFromTemplate([
-        ...(isUpdateAvailable() && [
-            {
-                label: 'update available',
-                click: () => showUpdateDialog(),
-            },
-        ]),
+        ...(isUpdateAvailable()
```

Comment:
why?

Reply:
It used to throw some error due to no default value, will reproduce it...